styling/Phoenix_files/_hero.scss

```diff
@@ -84,3 +85,25 @@ body, h1, h2, h3, h4, h5, h6 {
   background-image: url("../Phoenix_files/phoenix-logo-white.png") !important;
   background-size: 250px 34px;
 }
+
+#subheader-columns {
+  margin: 50px 30px 0; }
+
+.subheader h4 {
+  color: #000;
+  font-weight: $light-font;
+  font-size: 18px;
+  padding-bottom: 14px;
+  text-transform: uppercase; }
+
+.subheader p {
+  font-weight: $light-font;
+  color: #333;
+  font-size: 16px; }
+
+.subheader a {
```

Comment:
I think it's good to avoid nesting if we can. Less breakage when html changes. Since we have full control over this html maybe we could add a class to the link like `subheader-link` What do you think about that?